autogen/main/cluster.tf.tmpl

```diff
@@ -864,6 +864,14 @@ resource "google_container_node_pool" "windows_pools" {
             gpu_driver_version = lookup(each.value, "gpu_driver_version", "")
           }
         }
+
+        dynamic "gpu_sharing_config" {
+          for_each = lookup(each.value, "gpu_sharing_strategy", "") != "" ? [1] : []
```

Comment:
Should `gpu_sharing_strategy` be documented here? https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/autogen/main/README.md?plain=1#L97